sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -92,11 +92,11 @@ private static string GetDefaultScope(Uri uri)
             => $"{uri.GetComponents(UriComponents.SchemeAndServer, UriFormat.SafeUnescaped)}/.default";
 
         /// <summary>
-        /// Creates a <see cref="ConfigurationSetting"/> if the setting, uniquely identified by key and label, does not already exist in the configuration store.
+        /// Creates a <see cref="ConfigurationSetting"/> if the setting, uniquely identified by keyFilter and labelFilter, does not already exist in the configuration store.
```

Comment:
The parameter names on these methods are still key and label - what is the reason to make these keyFilter and labelFilter?

Reply:
Autorename artifact :(
Will fix.